src/Classes/TradeQuery.lua

```diff
@@ -0,0 +1,897 @@
+-- Path of Building
```

Comment:
I haven't fully vetted the queries yet, but there is a lot that can be cleaned up about this file.  For starters, having separate classes for each API would be a huge boon to readability.  Mixing the UI elements with backend calls in one file is really hard to follow.

Secondly, there are better ways to display this data than in EditControls.  Simple text works fine and can be updated easily.  Maybe for the status indicator you can change the text of "Price Item" or have a tooltip pop-up over the disabled button, I'm not sure.  Having a random edit control at the top is confusing, though.